h2o-hadoop-2/h2o-mapreduce-2-generic/build.gradle

```diff
@@ -1,6 +1,7 @@
 ext {
   hadoopVersion = 'cdh5.2'
   hadoopMavenArtifactVersion = '2.5.0-cdh5.2.0'
+  hiveMavenArtifactVersion = '0.13.1-cdh5.2.0'
```

Comment:
I guess this is inspired by above ^^^ hadoop version. I think the issue is that `hadoopMavenArtifactVersion` is overridden in each assembly but `hiveMavenArtifactVersion` is defined just here. We should override the definition on the assembly level for each supported distribution.

Reply:
I dont think one project can override ext in another so these stay exactly the same and this project is always compiled against these libs

Reply:
It used to work that way (until last December)... if it doesn't anymore I think that might be an issue. Let me check